dockerfile/parser/testfiles/health/Dockerfile

```diff
@@ -7,4 +7,6 @@ HEALTHCHECK --interval=5s --timeout=3s --retries=3 \
 HEALTHCHECK CMD
 HEALTHCHECK   CMD   a b
 HEALTHCHECK --timeout=3s CMD ["foo"]
+HEALTHCHECK --start-interval=5s --interval=5s --timeout=3s --retries=3 \
```

Comment:
This one could use a "--start-period" flag, since it's exercising the parser, and I don't see anything else in this directory checking that the flag is recognized.

Reply:
Done.